### PR TITLE
Add Django version 1.11.20

### PR DESCRIPTION
**Pull request by astamminger**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+  sha256: 43a99da08fee329480d27860d68279945b7d8bf7b537388ee2c8938c709b2041
   patches:
     # Hard-code gdal and geos' paths to the corresponding conda packages.
     - libgdal.patch
     - libgeos.patch
 
 build:
   number: 0
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  script: python -m pip install --no-deps --ignore-installed .
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Please see https://conda-forge.org/docs/maintainer/adding_pkgs.html#use-pip for the right line incantation we should be using here.

**Comment by ocefpaf** [comment]
Windows is failing with:

```
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7.
Processing c:\bld\django_1552505269515\work
Could not install packages due to an EnvironmentError: [('C:\\bld\\django_1552505269515\\work\\tests\\staticfiles_tests\\apps\\test\\static\\test\\\xe2??.txt', 'c:\\users\\appveyor\\appdata\\local\\temp\\1\\pip-req-build-0cmsif\\tests\\staticfiles_tests\\apps\\test\\static\\test\\\xe2??.txt', "[Errno 22] invalid mode ('rb') or filename: 'C:\\\\bld\\\\django_1552505269515\\\\work\\\\tests\\\\staticfiles_tests\\\\apps\\\\test\\\\static\\\\test\\\\\\xe2??.txt'")]
````

We can either remove that "offending" test file, I don't think it will be needed in a django installation, or skip Python 2.7 on Windows entirely if you are not worried about that platform.

**Comment by astamminger** [comment]
I wonder why it is working for python 3.6/3.7 on windows

**Comment by ocefpaf** [comment]
> I wonder why it is working for python 3.6/3.7 on windows

It is an annoying Windows thing where utf-8 characters create these sort of problem when on the path. There is probably a solution somewhere, just not sure if it is worth fixing it at this point. (I'm not fixing any legacy python failures in 2019 unless I really need to :smile:)

**Comment by astamminger** [comment]
@ocefpaf hey, let's just skip python 2.7 on windows for the time being if you're fine with that. I'll come back to fixing this issue as soon as I can make up some time (probably next week)

**Pull request merged**